resources/js/app.js

```diff
@@ -122,7 +122,7 @@ if (typeof io !== 'undefined') {
 Vue.toasted.register('showError',
     (payload) => {
         if(! payload.message) {
-    	    return "Maaf, telah terjadi sesuatu<br>(Panggil FAI / AND untuk dilihat lebih lanjut)"
+    	    return "Maaf, telah terjadi sesuatu<br>(Panggil AND / AMF untuk dilihat lebih lanjut)"
```

Comment:
Ah I see, you are a man of culture as well, keep doing this as our heritage...
![image](https://acegif.com/wp-content/gifs/tea-19.gif)

Reply:
> Well done today @adhiiimf Everything is just clear... Next, we need to fix the deployment script

thankyouu @IritaSee , yeap that's deployment script is so .. wkwkwk